members/FMNS420b9c88.yaml

```diff
@@ -65,7 +65,7 @@ contact_form:
           Senator: Senator
       - name: topic
         selector: select#edit-submitted-message-details-topic
-        value: $TOPIC
+        value: "$TOPIC"
         required: true
```

Comment:
`options`?

Reply:
TOPIC options are below. Should I move them above `required: true` ?

Reply:
TOPIC options are below. Should I move them above `required: true` ?

Reply:
didn't see that, sorry!